src/scripts.js

```diff
@@ -130,20 +134,37 @@ window.addEventListener("load", async () => {
     });
   });
 
-  [...Array(days)].forEach((_, i) => {
+  function createNavItem({ label = "", classes = [], onClick = () => {} }) {
```

Comment:
using `render` might simplify some things?

Reply:
Good idea. We'd still have to attach the handler manually though.